src/core/useGLTF.tsx

```diff
@@ -16,10 +19,15 @@ function extensions(useDraco: boolean | string, useMeshopt: boolean) {
   }
 }
 
-export function useGLTF(path: string, useDraco: boolean | string = true, useMeshOpt: boolean = true) {
-  const gltf = useLoader(GLTFLoader, path, extensions(useDraco, useMeshOpt))
+export function useGLTF(
+  path: string,
+  useDraco: boolean | string = true,
+  useMeshOpt: boolean = true,
+  extendLoader: any
```

Comment:
This shouldn't be any. This should be a function type. I'm assuming it returns void?

Reply:
@joshuaellis Yes, it returns void. Fixed the ts.